### PR TITLE
fix: nested CODE tags

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 1]
```diff
@@ -373,22 +373,21 @@ <h3 id="Listing_CRLs_in_a_Database"><code> <a id="1034026"> Listing CRLs in a Da
...
+<h3 id="Modifying_CRL_in_a_Database">Modifying CRL in a Database</h3>
+<p>This example modifies a new CRL and importing it in to a Database in the specified directory:</p>
+<p><code>crlutil -G -d </code><i>certdir</i><code> -n </code><i>cert-nickname</i><code> -c </code><i>crl-script-file</i></p>
+<p>or</p>
+<p><code>crlutil -M -d </code><i>certdir</i><code> -n </code><i>cert-nickname</i><code> &lt;&lt;<i>EOF<br>
   update=20050204153000Z<br>
   addcert 40-60 20050105153000Z<br>
-  EOF</i></code></code></p>
-<p><code>The CRL Management Tool extracts existing CRL from a database, will modify and sign with certificate <i>cert-nickname</i> and will store it in database. To modify while importing CRL from file user should supply <code>-i </code><i>import-crl-file</i> option. </code></p>
+  EOF</i></code>/p>
```
Comment (bershanskiy) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:391]: There might be missing `<` in `</p>`.

Comment (nschonni) [reply files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:391]:
```suggestion
  EOF</i></code></p>
```

File: files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html [hunk 2]
```diff
@@ -277,7 +277,7 @@ <h3 id="Up-mixing_and_down-mixing">Up-mixing and down-mixing</h3>
     <code>output.L = input.L + 0.7071 * input.C<br>
     output.R = input.R + 0.7071 * input.C<br>
     <code>output.SL = input.SL<br>
```
Comment (bershanskiy) [inline files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html:279]: `<code>` on line 279 is unnecessary, especially after removal of `</code>` on line 280.

Comment (nschonni) [reply files/en-us/web/api/web_audio_api/basic_concepts_behind_web_audio_api/index.html:279]:
```suggestion
    output.SL = input.SL<br>
```

File: files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html [hunk 3]
```diff
@@ -351,18 +351,18 @@ <h3 id="Creating_a_New_CRL"> <a id="1028724"> Creating a New CRL </a></h3>
...
-<h3 id="Listing_CRLs_in_a_Database"><code> <a id="1034026"> Listing CRLs in a Database </a></code></h3>
-<p><code><a id="1034749"> This example lists all the CRLs in the <code>NSS database</code> in the specified directory: </a></code></p>
-<p><code><a id="1034751"> <code>crlutil -L -d </code><i>certdir</i> </a></code></p>
-<p><code><a id="1034753"> The CRL Management Tool displays output similar to the following: </a></code></p>
-<p><code><a id="1034755"> <code>CRL Name              CRL Type</code></a></code></p>
-<p><code><a id="1037237"> <code>CN=NSS Test CA,O=BOGUS NSS,L=Mountain View,ST=California,C=US  CRL<br>
-  CN=John Smith,O=Netscape,L=Mountain View,ST=California,C=US  CRL</code></a></code></p>
-<p><code>To view a particular CRL user should specify <i>-n nickname</i> parameter.<br>
-  <code>crlutil -L -d </code><i>certdir</i><code> -n </code><i> nickname</i></code></p>
+<p><code>Where <i>cert-nickname</i> is the name the new CRL will be signed with.</p>
```
Comment (bershanskiy) [inline files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:354]: This whole file is spaghetti of misused tags. To make this document a valid HTML, please consider removing `<code>` on line 354.

On a side note, I in a follow-up PR, I'll change all these CLI commands and output examples to `<pre>`.

Comment (nschonni) [reply files/en-us/mozilla/projects/nss/tools/nss_tools_crlutil/index.html:354]:
```suggestion
<p>Where <i>cert-nickname</i> is the name the new CRL will be signed with.</p>
```

**Comment by bershanskiy** [comment]
@nschonni Thank you for addressing all my commits so promptly! This LGTM now.

**Review by chrisdavidmills — APPROVED** [review]
Cool, thanks @nschonni, let's get this merged.

And thanks to @bershanskiy for the review.

**Pull request merged**